### PR TITLE
Replace figures in docs/feature_split_keyboard.md

### PR DESCRIPTION
**Pull request by mtei**
## Description

Replace figures in docs/feature_split_keyboard.md

sk-pd0-connection-mono
sk-pd2-connection-mono

sk-i2c-connection-mono

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

**Review by drashna — APPROVED** [review]
**slams approve**

File: docs/feature_split_keyboard.md [hunk 1]
```diff
@@ -48,19 +48,20 @@ However, USB cables, SATA cables, and even just 4 wires have been known to be us
 
 ### Serial Wiring
 
-The 3 wires of the TRS/TRRS cable need to connect GND, VCC, and D0 (aka PDO or pin 3) between the two Pro Micros. 
+The 3 wires of the TRS/TRRS cable need to connect GND, VCC, and D0/D1/D2/D3 (aka PDO/PD1/PD2/PD3) between the two Pro Micros.
```
Comment (fauxpark) [inline docs/feature_split_keyboard.md:51]:
```suggestion
The 3 wires of the TRS/TRRS cable need to connect GND, VCC, and D0/D1/D2/D3 (aka PD0/PD1/PD2/PD3) between the two Pro Micros. 
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: docs/feature_split_keyboard.md (hunk 1 above)
Comment (drashna) [reply docs/feature_split_keyboard.md:51]: nice catch

**Comment by drashna** [comment]
And can confirm that it renders correctly on the doc site.

**Pull request merged**